sci_strategic_plugin/src/sci_strategic_plugin.cpp

```diff
@@ -394,19 +571,19 @@ cav_msgs::Maneuver SCIStrategicPlugin::composeLaneFollowingManeuverMessage(int c
 
   std::vector<double> float_metadata_list;
 
-  double speed_before_decel = calcSpeedBeforeDecel(time_to_stop, end_dist, start_speed);
+  double speed_before_decel = calcSpeedBeforeDecel(time_to_stop, end_dist-start_dist, start_speed);
 
   switch(case_num)
   {
     case 1:
       caseOneSpeedProfile(speed_before_decel, start_speed, time_to_stop, &float_metadata_list);
       break;
     case 2:
-      caseTwoSpeedProfile(end_dist, speed_before_decel, start_speed, time_to_stop, speed_limit_, &float_metadata_list);
-      break;
-    case 3:
-      caseThreeSpeedProfile(end_dist, start_speed, time_to_stop, &float_metadata_list);
+      caseTwoSpeedProfile(end_dist-start_dist, speed_before_decel, start_speed, time_to_stop, speed_limit_, &float_metadata_list);
```

Comment:
Is case 3 intended to be running the case 2 speed profile? It's unclear, and the dead code below indicates possibly otherwise?

Reply:
No case 3 is moved and is planned by the stop and wait plugin since it is only deceleration to stop. I think github is showing the lines incorrectly.